src/content/docs/en/pages/products/edge-firewall/rules-engine.mdx

```diff
@@ -66,7 +66,8 @@ These are all the Criteria variables you can set:
 | Request Method                | The request's HTTP method. For example: `GET`, `POST`, `PUT`, etc. | -                        |
 | Request URI                   | This relates to the `${uri}` variable from Rules Engine for Edge Applications. The normalized (urldecoded) URI of the request. The value of `${uri}` can change during the processing of a request, for example, when an internal redirect occurs or when index files are used. It doesn't carry the Query String parameters as `${request_uri}` do.| -                        |
 | Scheme                        | The scheme of the request: HHTP or HTTPS.                    | -                        |
-| Client Certificate Validation | Server process to authenticate client digital certificate. | - |  
+| Client Certificate Validation | Server process to authenticate client digital certificate. | - | 
+| SSL Verification Status       | Server result to client certificate validation. It can be `SUCESS`, when the client certificate validation pass; `CERTIFICATE_VERIFICATION_ERROR`, when the client certificate was not valid; and `MISSING_CLIENT_CERTIFICATE`, when the client certificate was not sent in the request. | - |  
```

Comment:
```suggestion
| SSL Verification Status       | Server result to client certificate validation. It can be `SUCCESS`, when the client certificate validation pass; `CERTIFICATE_VERIFICATION_ERROR`, when the client certificate was not valid; and `MISSING_CLIENT_CERTIFICATE`, when the client certificate was not sent in the request. | - |  
```